src/plugins/centos.py

```diff
@@ -1,7 +1,7 @@
 import re
 
 distribution = "centos"
-abrtparser = re.compile(r"^CentOS Linux release ([0-9]+) \(([^\)]+)\)$")
+abrtparser = re.compile(r"^CentOS Linux release (\d+)(?:\S(\d+)\S(\d+))? \(([^\)]+)\)$")
 guessparser = re.compile(r"\.el([0-9]+)")
```

Comment:
How about using `\d` on this line too, instead of `[0-9]`?

Reply:
Sure.